gisMetadataProcessing.py

```diff
@@ -0,0 +1,81 @@
+from bs4 import BeautifulSoup
```

Comment:
I would actually get rid of BeautifulSoup as a dependency and just use lxml directly. BeautifulSoup is really for parsing HTML. In this case, you should have well-formed FGDC XML. And if it isn't well-formed, you will want to know sooner rather than later.

Reply:
Good to know, I'll work on this

Reply:
So much more efficient! Thank you for pointing this out!